Carbon.Domain.EntityFrameworkCore.Extensions/Repositories/TenantManagedEFRepositoryBase.cs

```diff
@@ -81,9 +81,21 @@ public override async Task RemoveSolutions<T>(T relatedEntity)
             await TargetErDbContext.SaveChangesAsync();
         }
 
+        public override async Task RemoveSolutions<T>(List<T> relatedEntities)
+        {
+            if (relatedEntities == null || !relatedEntities.Any())
+                return;
+            var filteredIds = relatedEntities.Where(x => x.Id != Guid.Empty).Select(x => x.Id).ToList();
+            var entitySolutionRelations = await TargetErDbSet
+                .Where(k => !k.IsDeleted && k.EntityCode == relatedEntities.FirstOrDefault().GetObjectTypeCode() && filteredIds.Contains(k.EntityId))
+                .ToListAsync();
```

Comment:
relatedEntities.FirstOrDefault().GetObjectTypeCode() this can be problem; we have Ids but ObjectTypeCode can differ and this affects filter. Actually we don't need the ObjectTypeCode so we can remove this check from filter

Reply:
removed objectype filter